documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,84 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers Container.
```

Comment:
@rcaballeromx I agree with @jcvenegas that "The following steps describe how to get a bootchart from a Clear Containers container." is fine.